### ISSUE TITLE
User can't access summary page solely based on being an organization owner  

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#3645]
To reproduce, add the following property:

```xml
<property as="xs:string" name="oxf.fb.permissions.role.always-show">
    ["Organization Owner"]
</property>
```

Then create a form with the following permissions:

<img width="488" alt="screen shot 2018-07-02 at 1 36 34 pm" src="https://user-images.githubusercontent.com/106961/42185242-025fcf78-7dfd-11e8-977e-762000f8962f.png">

As "Jony Ive" (setting `My-Credentials-Header` to the JSON below), open the `/new` page and save.

```json
{
  "username" : "Jony Ive",
  "roles" : [
    { "name": "Chief Design Officer"                         },
    { "name": "Organization Owner", "organization": "Design" }
  ],
  "organizations" : [
    [ "Apple", "Design" ]
  ]
}
```

Then as "Tim Cook" (setting `My-Credentials-Header` to the JSON below), go to the summary page, and notice Tim gets a 403, while we would like Tim to be able to see the data entered by Jony.

```json
{
  "username" : "Tim Cook",
  "roles" : [
    { "name": "Organization Owner", "organization": "Apple" }
  ],
  "organizations" : [
    [ "Apple" ]
  ]
}
```

[+1 from customer](https://3.basecamp.com/3600924/buckets/2010455/messages/1159699220)

**Comment by avernet** [comment]
In the implementation of the persistence API, we just don't return anything if users aren't authorized solely based on their role, if their owners or group-member. We need to also exclude the case where users are authorized based on an organization match. This part is fixed in c79f90c28d0f3b2a7dbb10914548282bdb8b63ff.

**Comment by avernet** [comment]
Also removing the code in `fr/summary/view.xhtml` that tries to figure if the user anyway can't access the summary page, and returns a 403 in that case. This is redundant with code in the backend, creates code duplication, and there isn't huge benefit in returning a 403 vs. returning an empty summary page.

**Comment by avernet** [comment]
This has been fixed for 2018.1, backported to customer's 2016.3.1 branch, and [confirmed by customer](https://3.basecamp.com/3600924/buckets/2010455/messages/1159699220#__recording_1183195562).
